tedana/combine.py

```diff
@@ -79,6 +88,16 @@ def _combine_paid(data, tes):
     -------
     combined : (M x T) :obj:`numpy.ndarray`
         Data combined across echoes according to SNR/signal.
+
+    References
+    ----------
+    * Poser, B. A., Versluis, M. J., Hoogduin, J. M., & Norris,
```

Comment:
Should we just number all the references, as in `make_optcom` ?

Reply:
We don't have a Notes section for this function, so the reference number wouldn't be pointing to anything, which raises a warning during the build. We _could_ put the reference number in the first part of the docstring, but in cases where that first description appears in a table like the ones in the API page, _that_ raises a warning. This isn't a real problem for this docstring, since it's a private function and won't be built on the site, but I think it's a good pattern to get into.

Reply:
Ah, I see. Sorry though -- which "good pattern to get into" did you mean ?

Reply:
Just not numbering references when they'll produce a build warning.